opensourceleg/hardware/sensors.py

```diff
@@ -223,14 +223,14 @@ def initialize(self, number_of_iterations: int = 2000) -> None:
             self._zeroed = True
             self._log.info(f"[{self.__repr__()}] Zeroing routine complete.")
 
-        elif (
-            input(
-                f"[{self.__repr__()}] Would you like to re-initialize loadcell? (y/n): "
-            )
-            == "y"
-        ):
+        elif reset:
             self.reset()
-            self.initialize()
+            self.calibrate()
```

Comment:
Is this not recursive? Aren't we in the definition of self.calibrate??

Reply:
[65ec987](https://github.com/neurobionics/opensourceleg/pull/236/commits/65ec987f4d30ada9143e6a20a3f8d74fef6858e2)

Reply:
Lol fancy. Seems good.